src/cryptography/sign.js

```diff
@@ -24,8 +25,35 @@ const signatureHeader = createHeader('SIGNATURE');
 const secondSignatureHeader = createHeader('SECOND SIGNATURE');
 const signatureFooter = createHeader('END LISK SIGNED MESSAGE');
 
-export const signMessageWithPassphrase = (message, passphrase) => {
+const SIGNED_MESSAGE_HEADER = Buffer.from('Lisk Signed Message:\n', 'utf8');
```

Comment:
Here we call this `HEADER` but below we call it `prefix`. We should be consistent. Since we already have a "Header" in the presentation function, I have a preference for `SIGNED_MESSAGE_PREFIX`.